### PR TITLE
add example code use_two_plugins

### PR DESCRIPTION
**Pull request by xinyu3ru**
* add example code use_two_plugins.
* update README.MD file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,19 +31,124 @@ You are welcome to send your plugin to our contrib by creating a Pull Request!
...
+| 2 | DailyWords | [@markoxu](https://github.com/markoxu) | Say something everyday, like `Daily Words`. |
+| 3 | Rasa_Rest | [@markoxu](https://github.com/markoxu) | Rasa server channnel connector. |
+| 4 | Scheduler | [@wj-Mcat](https://github.com/wj-Mcat) | Say something everyday, like `Scheduler Words`. |
+| 5 | Room_inviter | [@markoxu](https://github.com/markoxu) | Invite peaple to room according keywords and rules then say hello words. |
+| 6 | Auto_reply | [@markoxu](https://github.com/markoxu) | AutoReply according keywords. |
+| 7 | Chat_history | [@markoxu](https://github.com/markoxu) | Save chat history with files and database. |
+| 8 | Github_webhook | [@wj-Mcat](https://github.com/wj-Mcat)| Github webhook, hook infomation to peaple or room when projects were updated. |
+| 9 | GitlabEvent | [@wj-Mcat](https://github.com/wj-Mcat) | Gitlab webhook, send gitlab event infomation to peaple or room when projects were updated. |
+| 10 | Finders | [@wj-Mcat](https://github.com/wj-Mcat) | Example code How to find room or contact, then do something. |
+| 12 | Use_two_plugins | [@xinyu3ru](https://github.com/xinyu3ru) | Example code How to use two or more plugins. |
```
Comment (wj-Mcat) [inline README.md]: How did you get the author of this plugin ?  There are many mistake in author column.  Please correct it.

File: README.md [hunk 2]
```diff
@@ -31,19 +31,124 @@ You are welcome to send your plugin to our contrib by creating a Pull Request!
...
+- Description: Example code How to find room or contact, then do something.
+
+Example code in examples folder, [Example of Finders](https://github.com/wechaty/python-wechaty-plugin-contrib/blob/master/examples/finders.py)
+
+
+### 12 Use_two_plugins
+- Author: [@xinyu3ru](https://github.com/xinyu3ru)
+- Description: Example code How to use two or more plugins. 
+
+Example code in examples folder, [use_two_plugins](https://github.com/wechaty/python-wechaty-plugin-contrib/blob/master/examples/use_two_plugins.py)
```
Comment (wj-Mcat) [inline README.md]: Please get the author of plugin from the pull request or forced commit. Or you can see the history of plugin file not the example code file.

File: README.md [hunk 3]
```diff
@@ -68,6 +178,7 @@ The `python-wechaty-plugin-contrib` project was created.
 
 - @wj-Mcat - [wj-Mcat](https://github.com/wj-Mcat), nlp researcher
 - @huan - [Huan LI](https://github.com/huan) ([李卓桓](http://linkedin.com/in/zixia)), Tencent TVP of Chatbot
+- @markoxu - [@markoxu](https://github.com/markoxu), Software Development Engineer at Intel Corporation.
```
Comment (wj-Mcat) [inline README.md:177]: `markoxu` is the contributor not the maintainer.  Please remove it from the docs.

File: examples/use_two_plugins.py [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+        await administrator.say(log_msg)
+
+ 
+    async def on_login(self, contact: Contact):
+        """login event
+        Args:
+            contact (Contact): the account logined
+        """
+        logger.info('微信号<%s> 已经登录……', contact)
+        self.login_user = contact
```
Comment (wj-Mcat) [inline examples/use_two_plugins.py]: we should mainly add the plugin usage code not the usages of wechaty interface.  Please remove it and add more plugin example code.

**Review by wj-Mcat — CHANGES_REQUESTED** [review]
Thank for your improvement about the plugin module. Plase read my comment to fix mistakes.

File: examples/use_two_plugins.py (hunk 4 above)
Comment (xinyu3ru) [reply examples/use_two_plugins.py]: I removed this file.

File: README.md (hunk 3 above)
Comment (xinyu3ru) [reply README.md:177]: I corrected the maintainer and contributer.

File: README.md (hunk 2 above)
Comment (xinyu3ru) [reply README.md]: pls check my new pull request.

**Comment by wj-Mcat** [comment]
@xinyu3ru there are some conflict errors in the `README.md` file, please fix it later. Looking forward for your great next work.

**Pull request closed without merging**